site/src/main/java/uk/nhs/digital/ps/beans/Dataset.java

```diff
@@ -41,10 +41,20 @@ public String getGeographicCoverage() {
         return getProperty("hippotaxonomy:keys");
     }
 
-    @HippoEssentialsGenerated(internalName = "publicationsystem:Files")
+    /*
```

Comment:
Doesn't the migration mean we don't need to do this?

Reply:
There is a window when site is already using new code, but the JCR content is not migrated.

Reply:
Cool. Could you add a comment to the appropriate files to make this clear.